### PR TITLE
More 23w03a stuff

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -923,6 +923,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 1 tickDelta
 	METHOD method_46395 extinguishWithSound ()V
 	METHOD method_46396 reinitDimensions ()V
+	METHOD method_48155 canSprint ()Z
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/Entity.mapping]: Does this not collide with `method_46743 canSprint ()Z` in `ClientPlayerEntity`?

Comment (apple502j) [reply mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
```

File: mappings/net/minecraft/text/Text.mapping [hunk 2]
```diff
@@ -62,6 +62,13 @@ CLASS net/minecraft/class_2561 net/minecraft/text/Text
 		COMMENT {@return whether the text contains {@code text}, without considering styles}
 		ARG 1 text
 	METHOD method_44746 withoutStyle ()Ljava/util/List;
+	METHOD method_48321 translatableWithFallback (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/class_5250;
+		ARG 0 key
+		ARG 1 fallback
+	METHOD method_48322 translatableWithFallback (Ljava/lang/String;Ljava/lang/String;[Ljava/lang/Object;)Lnet/minecraft/class_5250;
```
Comment (Shnupbups) [inline mappings/net/minecraft/text/Text.mapping:68]: Could these not just be `translatable`? I feel the `fallback` argument name is enough to get that these are ones with fallback.

**Review by Shnupbups — APPROVED** [review]

File: mappings/net/minecraft/text/Text.mapping (hunk 2 above)
Comment (apple502j) [reply mappings/net/minecraft/text/Text.mapping:68]: This is for user convenience - to allow calling `translatable(String, ...Object)` without weird casts.

Comment (Shnupbups) [reply mappings/net/minecraft/text/Text.mapping:68]: Ah, yeah that makes sense.

**Review by Shnupbups — APPROVED** [review]

**Pull request merged**